frontend/src/store/actions/index.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable prettier/prettier */
+import axios from 'axios';
+export const REGISTER_USER = 'REGISTER_USER';
+export const LOGIN_USER = 'LOGIN_USER';
+export const LOGOUT_USER = 'LOGOUT_USER';
+export const ADD_ARTICLE = 'ADD_ARTICLE';
+export const DELETE_ARTICLE = 'DELETE_ARTICLE';
+export const FETCH_DATA = 'FETCH_DATA';
+export const DATA_FETCHED = 'DATA_FETCHED';
+export const ERROR = 'ERROR';
+
+export const registerUser = user => {
+	return dispatch => {
+		dispatch({ type: FETCH_DATA });
+		axios
+			.post('https://anywhere-reader-test.herokuapp.com/api/users', user)
+			.then(response =>
+				dispatch({
+					type: REGISTER_USER,
+					payload: {
+						//the payload you're giving the API to populate the new user
+					}
+				})
+			)
+			.then(
+				// Re-GET all the notes, with the newly added one included
+				axios
```

Comment:
Why are you getting notes for a newly registered user? Are we providing a default note with a general user guide or something?